components/form-toggle/package.json

```diff
@@ -5,10 +5,21 @@
   "license": "EUPL-1.2",
   "name": "@utrecht/form-toggle-css",
   "files": [
-    "dist/"
+    "dist/",
+    "src/"
   ],
-  "scripts": {},
-  "devDependencies": {},
+  "main": "dist/index.css",
+  "scripts": {
+    "build": "rollup -c ../rollup.config.mjs",
+    "clean": "rimraf dist"
+  },
+  "devDependencies": {
+    "@utrecht/focus-ring-css": "workspace:*",
+    "rollup": "3.29.4"
+  },
+  "dependencies": {
```

Comment:
I don't think we need the focus-ring-css as a dependency here as it's already a dev-dependency like everywhere else